packages/zoe/src/contracts/autoswap.js

```diff
@@ -64,7 +64,7 @@ export const makeContract = harden(zoe => {
             } else {
               return rejectOffer(inviteHandle);
             }
-            if (!amountMaths.Liquidity.isEmpty(proposal.want.Liquidity)) {
+            if (proposal.want.Liquidity !== undefined) {
```

Comment:
Isn't this equivalent to 
```
if (proposal.want.Liquidity) {
```

Reply:
I think that would test truthy or falsiness, which would be less precise. I want to ensure it's undefined, but `if (proposal.want.Liquidity) {` would not throw if proposal.wantLiquidity is anything falsy: false, 0, -0, 0n, "", null, undefined, and NaN.